src/requests/reviews.js

```diff
@@ -0,0 +1,23 @@
+import { GET, POST } from "./Helper";
+
+async function getReview({ user, applicationId, program }) {
+  return GET(`/api/reviews/${user.userId}/${applicationId}/${program}`);
+}
+
+async function getReviewCount({ userId, programId }) {
+  return GET(
+    `/api/reviews/${userId}/${programId}/?` +
+      new URLSearchParams({ count: true })
+  );
+}
+
+async function updateReview(databody) {
+  return POST("/api/reviews", databody);
+}
+
+//Used right now
+// async function getUserReviews(user, applicationId, programId) {
+//   const result = GET(`/api/reviews/${user.uid}`);
+// }
```

Comment:
If this is used right now, shouldn't we keep this for legacy usage?

Reply:
This file is only for appVersion 2.0 reviews. And it is un-used as it is for an admin page that i have yet to hook up